lsbfdump/src/main/scala/org/apache/daffodil/lsbfDump/LSBFDump.scala

```diff
@@ -0,0 +1,188 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.daffodil.lsbfDump
+
+import java.io.IOException
+import java.nio.ByteBuffer
+import java.nio.channels.FileChannel
+import java.nio.file.NoSuchFileException
+import java.nio.file.Paths
+
+object LSBFDump {
+
+  private val helpText: String =
+    """Usage: lsbfdump --file <filename> [--offset <offset>] [--length <numBytes>] [--noAddress] [--help]
+      |
+      |<filename>   : The file to read bytes from or '-' for standard input.
+      |[offset]     : The starting offset in the file (default is 0).
+      |[length]     : The number of bytes to display (default is 128).
```

Comment:
This wants to be usable with vast data files without issue. If you forget to pipe the output to more (or less) utility you will rapidly scroll thousands of lines (it's 4 bytes per line). 128 bytes is "a screen" of data roughly.

Reply:
I would consider that normal behavior for CLI tools. All the most common tools process the entire input and it's expected that users preprocess the data if they want something less. For example, to get the 128 byte behavior, you could instead do something like:

```
head -b 128 file.lsbf | lsbfdump
```

I'm not saying that we shouldn't have offset and length built-in (it's useful to avoid having to read the data twice), but I think that's the more expected behavior in the CLI world.

I could also imagine a user not realizing it truncates to 128 and not notice there are missing bytes. It's much easier to notice if more data is output than expected, press Ctrl-C and change the command, than it is to realize data was truncated to 128.